Create ui-c8y-1023-80-2-datapoint-list-widget-empty-after-update-to-latest-cockpit-version-11898.md (#4695)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1023-80-2-datapoint-list-widget-empty-after-update-to-latest-cockpit-version-11898.md

---
date: ""

> title: Data point list widget displays data correctly after cockpit update

Reviewer: ```suggestion
title: Data point list widget displays data correctly after application update
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-66738
version: 1023.80.2
---

> To maintain backward compatibility, the data points widget is not bound to the dashboard time context after an application update. This ensures that the date range remains unchanged.

Reviewer 2: ```suggestion
To maintain backward compatibility, the "Data point list" widget is not bound to the dashboard time context after an application update. This ensures that the date range remains unchanged.
```

---
date: ""

> title: Datapoint list widget empty after update to latest cockpit version (#11898) [GRAFT][release/cd] (#11965)

Reviewer: ```suggestion
title: Data point list widget displays data correctly after cockpit update
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-66738
version: 1023.80.2
---
# Backport

This will backport the following commits from `develop` to `release/cd`:
- [fix(Web SDK): [MTM-66738] Datapoint list widget empty after update to
latest cockpit version
(#11898)](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/11898)

<!--- Backport version: 9.5.1 -->

### Questions ?
Please refer to the [Backport tool
documentation](https://github.com/sorenlouv/backport)

[MTM-66738]:
https://cumulocity.atlassian.net/browse/MTM-66738?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

Co-authored-by: Lukasz Janusz <114157358+luja-c8y@users.noreply.github.com>

> Co-authored-by: Lukasz Janusz <lukasz.janusz@cumuloicty.com>

Reviewer: ```suggestion
To maintain backward compatibility, the data points widget is not bound to the dashboard time context after an application update. This ensures that the date range remains unchanged.
```
